tools/docker/mu-plugins/bypass_circular_refs_during_core_update.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Plugin Name: Bypass Circular Refs During Core Update
+ * Description: Some of our packages have circular refs for testing purposes, which makes the core upgrader choke. This removes the problematic call to `_upgrade_422_remove_genericons()`.
+ * Version: 1.0
+ * Author: Automattic
+ * Author URI: https://automattic.com/
+ * Text Domain: jetpack
+ *
+ * @package automattic/jetpack
+ */
+
+/**
+ * Remove problematic call to `_upgrade_422_remove_genericons()` from `wp-admin/includes/update-core.php`. That call
+ * recursively searches all directories for genericons files, which results in a circular reference loop for some
+ * of our projects.
+ *
```

Comment:
Should we mention that there's unfortunately no direct filter for the upgrade stuff, but it happens to call `wp_opcache_invalidate()` before requiring the file which gives us a chance to hack it?

Reply:
e5917f5